torch/csrc/autograd/python_variable.cpp

```diff
@@ -2036,4 +2038,21 @@ c10::Device concrete_device_fn(const c10::impl::PyInterpreter*, const c10::Tenso
   return toDevice(out.ptr());
 }
 
+c10::IntArrayRef concrete_strides_fn(const c10::impl::PyInterpreter*, const c10::TensorImpl* self) {
+  pybind11::gil_scoped_acquire gil;
+  at::impl::MaybeSetTLSOnEntryGuard guard;
+
+  auto out = torchDispatchFromTensorImpl(
+      self,
+      "stride",
+      py::module::import("torch")
+          .attr("ops")
+          .attr("aten")
+          .attr("stride")
```

Comment:
Don't think so, since `stride` without any op overloads doesn't exist? But `torch.ops.aten.stride` exists

```
>> torch.ops.aten.stride

<OpOverloadPacket(op='aten.stride')>
```